src/views/order.html

```diff
@@ -1,251 +1,332 @@
-                </div>
-                <small id="urlHelp" class="form-text text-muted">Only characters A-Z and 0-9</small>
-            </div>
+    </div>
 
-            <div id="avail" style="margin-top:30px;">
-              <label for="avail-input">High availibility?</label>
-              <div class="form-check">
-                <label class="form-check-label">
+    <!--URL, Availibilty, Sensitive Data, free or not -->
+    <div ng-show="orderModel.env">
+      <fieldset class="form-group">
+        <legend>Additional Information</legend>
+        <div id="url" ng-class="failedValidation.url ? 'has-danger' : 'class-if-false'">
+          <label for="url-input">Choose a prefix for your site's address, letters and numbers only.</label>
+          <div style="color:#d9534f" ng-show="validationModel.url===false">{{validationModel.url_reason}}</div>
+          <div class="form-inline" ng-class="validationModel.url===false ? 'has-danger' : 'class-if-false'">
+            <input type="text" ng-model="orderModel.url" ng-blur="validateUrl()" class="form-control" placeholder="Enter url">&nbsp; . &nbsp;
+            <input type="text" disabled class="form-control" placeholder="web.umich.edu"> &nbsp;&nbsp;<span ng-show="orderModel.url"> https://{{orderModel.url}}.web.umich.edu </span>
+          </div>
+        </div>
+
+        <div id="avail" style="margin-top:30px;" ng-class="failedValidation.avail ? 'has-danger' : 'class-if-false'">
+          <div class="form-check">
+            <p>Your environment can be set to scale with demand so that it will be highly available </p>
+            <label class="form-check-label">
                     <input type="checkbox" ng-model="orderModel.avail" class="form-check-input">
-                    Yes
+                    I want my site/app to be highly available.
                 </label>
-              </div>
-            </div>
+          </div>
+        </div>
 
-            <div id="itar" style="margin-top:30px;">
-              <label for="itar-input">Does your site contain sensitive data?</label>
-              <a href="https://www.safecomputing.umich.edu/dataguide/?q=home">Read more about the Umich Sensitive Data Guide</a>
+        <div id="itar" style="margin-top:30px;" ng-class="failedValidation.itar ? 'has-danger' : 'class-if-false'">
+          <label for="itar-input">Does your site contain sensitive data?</label>
+          <a href="https://www.safecomputing.umich.edu/dataguide/?q=home">Read more about the Umich Sensitive Data Guide</a>
 
-              <div class="form-check">
-                <label class="form-check-label">
-                    <input type="radio" ng-model="orderModel.itar" value="yes" class="form-check-input">
+          <div class="form-check">
+            <label class="form-check-label">
+                    <input type="checkbox" ng-model="orderModel.itar" class="form-check-input">
```

Comment:
Chris said that user has to attest that 'My Site Will Not Host Sensitive Data'. Maybe that's for legal purposes I'm not sure. Maybe that can show up on the preview request modal instead of 'No' so when they click submit they are confirming/attesting that their site will not host sensitive data.